samples/detect.v2beta1.js

```diff
@@ -0,0 +1,848 @@
+    projectId: projectId,
+  });
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = `your GCLOUD project name, if not specified as env variable e.g. my-Gcloud-Project`;
+  // const knowledgeBaseFullName = `the full path of your knowledge base, e.g my-Gcloud-project/myKnowledgeBase`;
+
+  client
+    .listDocuments({parent: knowledgeBaseFullName})
+    .then(responses => {
+      var resources = responses[0];
+      console.log(
+        `There are ${resources.length} documents in ${knowledgeBaseFullName}`
+      );
+      resources.forEach(resource => {
+        console.log(` KnowledgeType: ${resource.knowledgeType}`);
+        console.log(` displayName: ${resource.displayName}`);
+        console.log(` mimeType: ${resource.mimeType}`);
+        console.log(` contentUri: ${resource.contentUri}`);
+        console.log(` source: ${resource.source}`);
+        console.log(` name: ${resource.name}`);
+      });
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END dialogflow_list_document]
+}
+
+function getDocument(documentId) {
+  // [START dialogflow_get_document]
+  // Imports the Dialogflow client library
+  const dialogflow = require('dialogflow').v2beta1;
+
+  // Instantiate a DialogFlow Documents client.
+  var client = new dialogflow.DocumentsClient({
+    projectId: projectId,
+  });
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const documentId = `full path to document in knowledge base, e.g. myKnowledgeBase/documents/myDoc`;
+
+  client
+    .getDocument({name: documentId})
+    .then(responses => {
+      var r = responses[0];
+      console.log(` KnowledgeType: ${r.knowledgeType}`);
+      console.log(` displayName: ${r.displayName}`);
+      console.log(` mimeType: ${r.mimeType}`);
+      console.log(` contentUri: ${r.contentUri}`);
+      console.log(` source: ${r.source}`);
+      console.log(` name: ${r.name}`);
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END dialogflow_get_document]
+}
+
+function deleteDocument(projectId, documentId) {
+  // [START dialogflow_delete_document]
+  // Imports the Dialogflow client library
+  const dialogflow = require('dialogflow').v2beta1;
+
+  // Instantiate a DialogFlow Documents client.
+  var client = new dialogflow.DocumentsClient({
+    projectId: projectId,
+  });
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = `your GCLOUD project name, if not specified as env variable e.g. my-Gcloud-Project`;
+  // const documentId = `full path to document in knowledge base, e.g. myKnowledgeBase/documents/myDoc`;
+
+  client
+    .deleteDocument({name: documentId})
+    .then(responses => {
+      var operation = responses[0];
+      return operation.promise();
+    })
+    .then(responses => {
+      if (responses[2] !== null) console.log(`document deleted`);
```

Comment:
What is the value of `responses[2]` when the document has been successfully deleted?

I would much rather see `if (responses[2] == 'SUCCESS')` or something

Reply:
❗️ **#BLOCKER**

Blocking because this is a public snippet and I don't want `!== null` to be our recommendation for Dialogflow users for how to check if a delete was successful